### PR TITLE
missing templates and js files in pip package

### PR DESCRIPTION
**Pull request by boydjohnson**
I updated the Manifest.in file to include the template and static directories that include the datetimefilter.html and add_on.js needed for the changeview.

### CODE REVIEW

**Comment by peterfarrell** [comment]
After this PR, we'll need to have the package re-released on PyPI. Currently, the admin functionality doesn't work due to the missing templates (because I didn't think of adding them to the Manifest file).

**Comment by boydjohnson** [comment]
The packaging wasn't working, but now it is. The change was adding package_data to setup.py.
In this pull request I also have solutions to 
[Lack of Docs on Admin](https://github.com/swisscom/cleanerversion/issues/88)
[Lack of unicode support](https://github.com/swisscom/cleanerversion/issues/86)
[History not showing](https://github.com/swisscom/cleanerversion/issues/87)
as well as the packaging problem where versions/templates/versions/datetimefilter.html and versions/static/js/addon.js weren't included in the package.

File: setup.py [hunk 1]
```diff
@@ -33,8 +33,9 @@
       author_email='engineering.sophia@swisscom.com',
       license='Apache License 2.0',
       packages=find_packages(exclude=['cleanerversion', 'cleanerversion.*']),
-      url='https://github.com/swisscom/cleanerversion',
+      url='https://github.com/boydjohnson/cleanerversion',
```
Comment (maennel) [inline setup.py]: Just wondering: what's the purpose of changing this URL? I think it's ok if I change it back..

**Comment by maennel** [comment]
@boydjohnson Thanks for the PR! There was an error, however, in the overridden ``history_view`` method. Perhaps it would make sense to work a bit on the test coverage... ;)

File: setup.py (hunk 1 above)
Comment (peterfarrell) [inline setup.py]: I think it was an oversight.

**Pull request merged**